Documentation/getting_started/2_understanding_the_code.md

```diff
@@ -0,0 +1,90 @@
+
+This file will go over the code that is getting created when you start a blank project. For help on creating a project please look at [Creating a New Project](getting_started/1_creating_a_new_project.md)
+
+**Using Statements**
+```c#
```

Comment:
> csharp works :D
> 
> tho indentation is still broken.

Well bugs in the SharpDoc conversion shouldn't block this PR.  We should create issues for that on their repo and i'll see if I can find time next month to fix it.